.travis-fixtures/exception_detection/global_enable_detection.output

```diff
@@ -0,0 +1,452 @@
+
+
+  1) property default failing_raise (VerifyExceptionDetectionTest)
+     test/verify_exception_detection_test.exs:36
+     Property Elixir.VerifyExceptionDetectionTest.property default failing_raise() failed. Counter-Example is:
+     [0]
+     
+     Counter example stored.
+     
+     PropCheck detected an error:
+     ** (RuntimeError) raise-fail
```

Comment:
@alfert This is what the output looks like.

Reply:
Looks good! 👍